src/Plotting/line_plots.jl

```diff
@@ -1,232 +1,175 @@
-rcParams = PyPlot.PyDict(PyPlot.matplotlib."rcParams")
-rcParams["lines.linewidth"] = 2. # Change linewidth
-
-function fill_past(m, ylims)
-    domain_idx = (t(m) .> m.present_year)
-    fill_between(
-        t(m)[.~domain_idx],
-        ones(size(t(m)[.~domain_idx])) * ylims[1] * 2.,
-        ones(size(t(m)[.~domain_idx])) * ylims[2] * 2.,
-        facecolor="b", alpha=0.1
-    )
-    ylim(ylims)
-    return
-end
+gr()
```

Comment:
This is already the default for Plots.jl, so it's better to remove it here, and let the user change it if they want to by adding `gr()` in their notebook.